src/librustc_driver/driver.rs

```diff
@@ -216,6 +216,13 @@ pub fn compile_input(sess: &Session,
         })??
     };
 
+    if sess.opts.debugging_opts.print_type_sizes {
```

Comment:
It just looks better there. Also, then you could have the type size list on the global tcx.